lib/account.js

```diff
@@ -56,6 +56,19 @@ const ACCOUNT_ENDPOINTS = {
             body: JSON.stringify(email),
         };
     },
+    getDeleteAccountAuth(password) {
+        return {
+            path: '/accounts/delete',
+            method: 'POST',
+            body: JSON.stringify(password),
```

Comment:
`password` is actually an object with the key `password`. Would make more sense to pass the password directly here and send `JSON.stringify({ password })`

Reply:
ok yep, makes sense